### PR TITLE
BZ1304659: File Explorer: Package is expanded only after second click

### PR DESCRIPTION
**Pull request by manstis**
See https://bugzilla.redhat.com/show_bug.cgi?id=1304659

### CODE REVIEW

**Comment by ederign** [comment]
@manstis how this solve the issue?

File: uberfire-widgets/uberfire-widgets-core/uberfire-widgets-core-client/src/main/java/org/uberfire/ext/widgets/core/client/tree/TreeItem.java [hunk 1]
```diff
@@ -87,7 +88,6 @@ public TreeItem( final Type type,
                         public void onClick( ClickEvent event ) {
                             if ( !isSelected ) {
                                 updateSelected();
-                                return;
```
Comment (manstis) [inline uberfire-widgets/uberfire-widgets-core/uberfire-widgets-core-client/src/main/java/org/uberfire/ext/widgets/core/client/tree/TreeItem.java]: @ederign Here solves the issue.. previously when the DOM element was clicked it first selected it and exited; now it selects it and opens it.

**Comment by ederign** [comment]
+1!

File: uberfire-widgets/uberfire-widgets-core/uberfire-widgets-core-client/src/main/java/org/uberfire/ext/widgets/core/client/tree/TreeItem.java (hunk 1 above)
Comment (ederign) [inline uberfire-widgets/uberfire-widgets-core/uberfire-widgets-core-client/src/main/java/org/uberfire/ext/widgets/core/client/tree/TreeItem.java]: thanks!

**Pull request closed without merging**